src/mcode-pilot/utils/recordToProps.js

```diff
@@ -56,15 +56,22 @@ export default function getProps(patient, condition) {
                 })(),
                 "reference": condition.getMostRecentHistologicalGrade()
             },
-            "stage": {
-                "display": "stage",
+            "clinical_stage": {
+                "display": "clinical stage",
```

Comment:
Does it matter if the display for the item in the UI says "clinical stage" vs "pathologic stage"? The latest designs just show "stage".  (cc: @rdingwell)